packages/backend/src/usagers/agrements/agrements.repository.ts

```diff
@@ -38,14 +39,18 @@ async function insertAgrementSejours(
   agrement: AgrementDto,
 ) {
   if (!agrement.agrementSejours?.length) return;
-
   for (const s of agrement.agrementSejours) {
+    const adresseId = s?.adresse?.id
+      ? s.adresse.id
+      : s?.adresse
+        ? await saveAdresse(client, s.adresse)
+        : null;
```

Comment:
double ternaire à éviter :

```suggestion
        let adresseId = bhe?.adresse?.id || null
        if (!adresseId && bhe?.adresse) {
          adresseId =  await saveAdresse(client, bhe.adresse);
        }
```